File: JJB/1126/이어_붙인_수.py
def solution(num_list):
    answer = 0
    odd=[]
    even=[]
    for i in num_list:
        if i%2==0:
            even.append(i)
        else:
            odd.append(i)
    
    odd = list(map(str, odd))
    even = list(map(str, even))
    
    odd = int(''.join(odd))
    even = int(''.join(even))
    answer = odd + even
    
    return answer

File: YSY/12_6/이어 붙인 수.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

int solution(vector<int> num_list) {
    int answer = 0;
    string even="";
    string odd="";
    for (int& i:num_list)
    {
        if (i%2==0)
        {
            even+=to_string(i);
        }
        else if (i%2==1)
        {
            odd+=to_string(i);
        }
    }
    answer=stoi(odd)+stoi(even);
    return answer;
}